lib/lib_rss.php

```diff
@@ -823,25 +823,17 @@ function errorMessageInfo($errorTitle, $error = '') {
 		$details = "<pre>{$details}</pre>";
 	}
 
+	header($_SERVER['SERVER_PROTOCOL'] . ' 500 Internal Server Error', true, 500);
```

Comment:
```suggestion
	header('HTTP 1.1 500 Internal Server Error', true, 500);
```

Our response is still HTTP 1.1 compliant, even if client is talking HTTP 2.0 or 3.0 and that gets translated if necessary by the Web server and/or proxy anyway

Reply:
Maybe that line should be moved to the `killApp()` function

Reply:
`killApp()` is not enough.

errorMessageInfo() is used in `kippApp()` and:
https://github.com/FreshRSS/FreshRSS/blob/edge/p/i/index.php#L65-L69